JuliaGPU/session1/Manav Singhal/q2.jl

```diff
@@ -0,0 +1,10 @@
+no=parse(UInt8,readline())
```

Comment:
```suggestion
no=parse(UInt64,readline())
```
So that large numbers can be handled.